### ISSUE TITLE
[Issue] won't spawn any vanilla creature (at least inside)

### ISSUE THREAD

**Issue by CoolLKKPS** [AudioKnight/Starlancer#33]
same as topic, if u need my modpack pls mention me in here

**Comment by CoolLKKPS** [comment]
and i also installed rosie moonpack, is that matter i think bc i saw both using the same mod creatures

**Comment by AudioKnight** [comment]
Sorry for the delayed response. I haven't had any other reports of this happening, so I'm inclined to say that it was caused by a different mod.

**Comment by CoolLKKPS** [comment]
might be.. uh well i shouldn't have slept at this point, but i have a fever about hours ago that causes my body clock to be abnormal, so will test that now

**Comment by CoolLKKPS** [comment]
![image](https://github.com/AudioKnight/Starlancer/assets/96377775/e8f24ded-7aa5-45fa-b78d-f7bb23a9bbc2)
@AudioKnight

**Comment by CoolLKKPS** [comment]
yep as what i expect, conflict with LethalCompanyVariables v2.3.2 by AMRV, other moons won't have this issue

**Comment by CoolLKKPS** [comment]
but if installed starlancer moonpack, this issue will affect all moons
